services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +46,46 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  BaseType_t send = xQueueSend(thermalMgrQueueHandle, event, 0); // 0 ticks to wait 
+  if (send != pdPASS) return ERR_CODE_INVALID_QUEUE_MSG/*failed case*/;
```

Comment:
you can just do `if(xQueueSend(thermalMgrQueueHandle, event, 0) != pdPASS)` instead of declaring a new variable